BaragonUI/app/models/Group.coffee

```diff
@@ -76,6 +76,7 @@ class Group extends Model
                 vex.dialog.buttons.NO
             ]
             callback: (data) =>
+                return if data is false
```

Comment:
Ah, that makes sense. I had assumed the issue was not enforcing the value in the form. 👍